base/strings/basic.jl

```diff
@@ -34,6 +34,7 @@ If you need to subsequently modify `v`, use `String(copy(v))` instead.
 """
 String(v::Array{UInt8,1})
 
+String(x::Symbol) = string(x)
```

Comment:
As far as I know, `Symbol`s are currently never freed, so this is safe. But I don't know if it's a good idea to depend on this behaviour. @stevengj